README.md

```diff
@@ -0,0 +1,46 @@
+[![Docker Image Version](https://img.shields.io/docker/v/sergii.ivanov/redis-commander?sort=date&label=Version)](https://hub.docker.com/r/sergii.ivanov/redis-commander/tags)
+[![Docker Image Size](https://img.shields.io/docker/image-size/sergii.ivanov/redis-commander?label=Image%20Size)](https://hub.docker.com/r/sergii.ivanov/redis-commander/tags)
+[![Docker Pulls](https://img.shields.io/docker/pulls/sergii.ivanov/redis-commander?label=Pulls)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+[![Docker Stars](https://img.shields.io/docker/stars/sergii.ivanov/redis-commander?label=Docker%20Stars)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+
+# Redis Commander
+
+Redis web management tool written in node.js
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
+
+# Spec
+
+## Environment variables
+
+No any variables
+
+## Expose ports
+
+* `tcp/39000` - redis-commander listening endpoint
+
+## Volumes
+
+* `/docker-images` - Root of the redis-commander repositories
+
+# Inside
+
+* redis-commander/redis-commander:19cd0c49f418
```

Comment:
Here we have to define Redis Commander version (not image)

Reply:
* redis-commander/redis-commander:latest
DIGEST:sha256:19cd0c49f418779fa2822a0496c5e6516d0c792effc39ed20089e6268477e40a
or remove DIGEST?